src/main/java/ru/skypro/courseWork/mapper/CommentMapper.java

```diff
@@ -28,6 +28,9 @@ public interface CommentMapper {
 
     @Named("authorImageToString")
     default String authorImageToString(User user) {
+        if (user.getImage() == null) {
+            return "/users/image/null";
```

Comment:
```suggestion
            return null;
```
в dto в image надо null вернуть, а не строку